Phonebook.Frontend/src/styles/overrides.scss

```diff
@@ -21,3 +21,20 @@ $config: mat-typography-config();
   position: relative;
   height: inherit;
 }
+
+//style for the contributors table
+table {
```

Comment:
Please write this into the component.
https://stackoverflow.com/questions/36265026/angular-2-innerhtml-styling